### DISCUSSION TITLE
"gitpappy" listed as a contributer on my repos when im pushing from vs code

### DISCUSSION THREAD

**Discussion by Seyaji** [community/community#23136] [New to GitHub]
<p>Not sure why but “gitpappy” is listed as a contributor on my repos so my contributions are not being counted, can someone help me solve this?<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/6/86999da06683f429dff51e2c6f8a06f0cd161ccf.jpeg" data-download-href="/uploads/short-url/jcJ6h5Q0NyyzhLfbtwYdnnIWXvh.jpeg?dl=1" title="Screenshot 2022-06-10 at 22.18.18" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/67058118/181075053-0111fdd7-06c2-4cb7-b1b5-61023d35bfa9.jpeg" alt="Screenshot 2022-06-10 at 22.18.18" data-base62-sha1="jcJ6h5Q0NyyzhLfbtwYdnnIWXvh" width="600" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/6/86999da06683f429dff51e2c6f8a06f0cd161ccf_2_600x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/8/6/86999da06683f429dff51e2c6f8a06f0cd161ccf_2_900x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/6/86999da06683f429dff51e2c6f8a06f0cd161ccf_2_1200x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/6/86999da06683f429dff51e2c6f8a06f0cd161ccf_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screenshot 2022-06-10 at 22.18.18</span><span class="informations">1804×1502 73.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by Akash1134** [comment]
<p>Welcome to the GitHub community, <a class="mention" href="/seyaji">@Seyaji</a> We’re happy you’re here. You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails–giving a few more details might help someone give you a nudge in the right direction. I’ve gone ahead and moved it for you. Good luck!</p>

**Comment by airtower-luna** [comment]
<p>You’ll have to look at the commits that are identified as theirs to figure this out, specifically the author and committer email addresses. GitHub uses those to identify who made a commit. Maybe there’s a typo somewhere? At least that’s a common cause for problems like this.</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>this is the <code>gitpappy</code> user being pointed at</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/2/a2a2c01016d5871289a6704b36c59f9a53a4360a.png" data-download-href="/uploads/short-url/ncK2xCMmwgvYZpT9nyGgbtHtnrY.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181075055-dc64b9b0-8ba5-4418-9c9d-25b664d908b6.png" alt="image" data-base62-sha1="ncK2xCMmwgvYZpT9nyGgbtHtnrY" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/2/a2a2c01016d5871289a6704b36c59f9a53a4360a_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/a/2/a2a2c01016d5871289a6704b36c59f9a53a4360a_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/a/2/a2a2c01016d5871289a6704b36c59f9a53a4360a_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/2/a2a2c01016d5871289a6704b36c59f9a53a4360a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 57.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>the user’s last activity was last 2018</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/7/176587d6074c95f712ce45a69a88034113038d4b.png" data-download-href="/uploads/short-url/3kYvD0zDtC8RE7Ro9uUygOgSAI3.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181075056-563439a4-6c20-4a8e-a44a-c3355ea37de5.png" alt="image" data-base62-sha1="3kYvD0zDtC8RE7Ro9uUygOgSAI3" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/7/176587d6074c95f712ce45a69a88034113038d4b_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/7/176587d6074c95f712ce45a69a88034113038d4b_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/1/7/176587d6074c95f712ce45a69a88034113038d4b_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/7/176587d6074c95f712ce45a69a88034113038d4b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 52.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>so it seems to me the email is being used by another user</p>
<p>oh no, I further checked it, the user always appear at the tab</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/b/4b0bebd0cd268d424304f59dc289f545e0bd5173.png" data-download-href="/uploads/short-url/aHTlMal6pz6KtaPQ9kxcP2wFJqX.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181075058-182bb2f3-d886-45df-a513-87ebb82aa19b.png" alt="image" data-base62-sha1="aHTlMal6pz6KtaPQ9kxcP2wFJqX" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/b/4b0bebd0cd268d424304f59dc289f545e0bd5173_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/b/4b0bebd0cd268d424304f59dc289f545e0bd5173_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/b/4b0bebd0cd268d424304f59dc289f545e0bd5173_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/b/4b0bebd0cd268d424304f59dc289f545e0bd5173_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 63 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>this is the other one</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/7/876f13e1302667a9cf21af0e8d209a66b1b2e4ab.png" data-download-href="/uploads/short-url/jk6rmjq9DKctdUnlgPNpFfyJYSf.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181075059-48fda2b4-959f-47e9-bd15-de50cc5dfd16.png" alt="image" data-base62-sha1="jk6rmjq9DKctdUnlgPNpFfyJYSf" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/7/876f13e1302667a9cf21af0e8d209a66b1b2e4ab_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/7/876f13e1302667a9cf21af0e8d209a66b1b2e4ab_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/7/876f13e1302667a9cf21af0e8d209a66b1b2e4ab_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/7/876f13e1302667a9cf21af0e8d209a66b1b2e4ab_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 60.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>so it seems to me, it’s an email conflict, either both of you uses it, the other user, in the past and you now, but still pointing to that</p>
<p>again, I’m not saying I’m correct, that’s just my assumption</p>

**Comment by Seyaji** [comment]
<p>Thanks everyone for the responses, i have reconfigured my git email and my contributions are now working :slight_smile:</p>
